src/pages/postgraphile/usage-library.md

```diff
@@ -137,6 +137,24 @@ create function get_x_something() returns text as $$
 $$ language sql stable;
 ```
 
+Everything returned by pgSettings is applied to the current session with ```set_config($key, $value, true)```
+
+Internal Postgres settings (keys without periods) can be changed directly such as 'role':
+
+```js
+export postgraphile(process.env.DATABASE_URL, schemaName, {
+  pgSettings: req => ({
+    'role': 'visitor',
+    'jwt.claims.user_id': `${req.user.id}`,
+    //...
+  }),
+})
+```
+
+```sql
+select current_user; -- returns visitor
+```
```

Comment:
This `sql` snippet may end up being confusing to people, because they won't know how to issue that command within the context of pgSettings. Might be better to create a custom query that they can then query from GraphQL?